src/genie/libs/parser/iosxr/show_rib.py

```diff
@@ -0,0 +1,200 @@
+""" show_rib.py
+    supports commands:
+        * show rib table
+        * show rib summary 
+"""
+
+# Python
+import re
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Optional
+
+# parser utils
+from genie.libs.parser.utils.common import Common
+
+# =============================================
+# Parser for 'show rib table'
+# =============================================
+
+class ShowRibTablesSchema(MetaParser):
+    schema = {'rib_tables': {
+                        'table_id':{
+                            Any(): {
+                                'prefix_count': int,
+                                'prefix_limit': int,
+                                'prefix_limit_notified': str,
+                                'safi': str,
+                                'table_deleted': str,
+                                'table_id': str,
+                                'table_name': str,
+                                'table_reached_convergence': str,
+                                'table_version': int,
+                                'vrf_name': str,
+                                'forward_referenced' : str,
+                        }
+                    }
+                }
+            }  
+
+class ShowRibTables(ShowRibTablesSchema):
+    """ Parser for show rib tables"""
+
+    
+    cli_command = ['show rib tables']
+
+    def cli(self,port="",output=None):
+        """parsing mechanism: cli
+        Function cli() defines the cli type output parsing mechanism which
+        typically contains 3 steps: exe
+        cuting, transforming, returning
+        """
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+            
+        result_dict = {}
+        
+        p1 = re.compile(r'^(?P<vrf>\S+)\/(?P<table>\w+) +(?P<safi>\w+)'
```

Comment:
Can you add an example for the regexp?
![image](https://user-images.githubusercontent.com/33664528/95133933-0c3bc380-0730-11eb-917c-4ccbdb3e0492.png)

Reply:
Sure, I can do that. Did you want an example per regex compile line for every single one or just that specific one (p1)? I assume you would like a (#) comment line with an example raw line as per what you have here?

Reply:
Hi, we usually do put an example per regexp so whoever is going to maintain/enhance in the future knows where exactly was the flaw in the regxp., thank you!